src/test/README.md

```diff
@@ -12,22 +12,23 @@ that runs all of the unit tests. The main source file for the test library is fo
 
 ### Compiling/running unit tests
 
-Unit tests will be automatically compiled if dependencies were met in `./configure`
+Unit tests will be automatically compiled if dependencies were met
+during the generation of the Bitcoin Core build system
 and tests weren't explicitly disabled.
 
-After configuring, they can be run with `make check`, which includes unit tests from
-subtrees, or `make && make -C src check-unit` for just the unit tests.
+Assuming the build directory is named `build`, the unit tests can be run
+with `ctests --test-dir build`, which includes unit tests from subtrees.
 
-To run the unit tests manually, launch `src/test/test_bitcoin`. To recompile
-after a test file was modified, run `make` and then run the test again. If you
-modify a non-test file, use `make -C src/test` to recompile only what's needed
+To run the unit tests manually, launch `build/src/test/test_bitcoin`. To recompile
+after a test file was modified, run `cmake --build build` and then run the test again. If you
+modify a non-test file, use `cmake --build build --target test_bitcoin` to recompile only what's needed
```

Comment:
Maybe I'm missing context, but whether test file was modified or production code, both `cmake --build build --target test_bitcoin` and `cmake --build build` will build all dependencies automagically, so maybe we can simply recommend `cmake --build build --target test_bitcoin` in all cases - and if we're worried about speed, we could even mention the `-j N` param here as well.

Reply:
Yes, ` cmake --build build --target test_bitcoin` always works.